source/runner/DangerfileRunner.ts

```diff
@@ -66,7 +67,7 @@ export async function dangerJestConfig() {
       defaultPlatform: "danger-js",
     },
     moduleNameMapper: [],
-    moduleDirectories: ["node_modules"],
+    moduleDirectories: [path.join(process.cwd(), "node_modules")],
```

Comment:
Looks like it doesn't - at least the results are actually the problem that I was having on Peril.